src/main/kotlin/no/nav/klage/oppgave/api/KlagebehandlingController.kt

```diff
@@ -31,25 +30,30 @@ class KlagebehandlingController(
 
     @GetMapping("/klagebehandlinger/{id}")
     fun getKlagebehandling(
-        @PathVariable("id") oppgaveId: Long,
-        request: HttpServletRequest
+        @PathVariable("id") klagebehandlingId: String
     ): KlagebehandlingView {
-        val innloggetIdent = innloggetSaksbehandlerRepository.getInnloggetIdent()
         logger.debug(
             "getKlagebehandling is requested by ident {} for oppgaveId {}",
-            innloggetIdent,
-            oppgaveId
+            innloggetSaksbehandlerRepository.getInnloggetIdent(),
+            klagebehandlingId
         )
-        return klagebehandlingFacade.getKlagebehandling(oppgaveId).also {
+        return klagebehandlingFacade.getKlagebehandling(klagebehandlingId.toUUIDOrException()).also {
             auditLogger.log(
                 AuditLogEvent(
                     navIdent = innloggetSaksbehandlerRepository.getInnloggetIdent(),
-                    action = KLAGEBEHANDLING_VIEW,
-                    decision = ALLOW,
+                    action = AuditLogEvent.Action.KLAGEBEHANDLING_VIEW,
+                    decision = AuditLogEvent.Decision.ALLOW,
                     personFnr = it.foedselsnummer
                 )
             )
         }
     }
 
+    private fun String.toUUIDOrException() =
+        try {
+            UUID.fromString(this)
```

Comment:
Mener jeg spurt om dette før (i annen PR), men kan vi ikke bruke UUID direkte som parameter i mottaket? Hvis jeg fikk svar da så husker jeg ikke..

Reply:
Jeg husker ikke om jeg svarte. Svaret er at jeg vet ikke. Men vi hadde en diskusjon for lange siden med hele teamet, og ble vel enige der om å bruke String. Og det er det jeg har forholdt meg til.. Vi kan gjerne ta en gjennomgang på dette senere.